content/apidokumentasjon/reelle-rettighetshavere/maskinell-innrapportering/beskrivelse-av-felter/_index.md

```diff
@@ -4,6 +4,66 @@ description: Tekstlig beskrivelse av feltene i JSON-skjemaet
 weight: 4
 ---
 
-{{< warning >}}
-Vi jobber med en tekstlig beskrivelse av feltene i JSON-skjemaet som kommer på denne siden.
-{{< /warning >}}
+
+Denne siden gir en komplett beskrivelse av feltene du finner i [JSON-skjema](https://schema.brreg.no/reelle/altinn/schema.json) for maskinell innrapportering. Tilhørende kodeliste finnes her i [denne Excel-filen](Kodeverk%20MINN.xlsx) 
+
+## Feltbeskrivelse
+
+
+| Klasse                                                                 | Egenskap                                                                                                                                            | Beskrivelse                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Kan endres? | Påkrevd  | Betingelser ved utfylling                                                                                                                                                                                                                                         |
```

Comment:
Ordet "klasse" blir vel feil her. Alt er vel egentlig properties/egenskaper. Men denne kan ikke hete det samme som neste kolonne som heter "Egenskap". Kanskje man skulle brukt "Property" som begrep men man må skille de to kolonnene. Det blir litt teknisk, men hva med "Property" og "Sub-property"? Evt noe tilsvarende.

Reply:
Jeg tar i bruk Propert/Sub-property på denne.